src/screens/params/utils.ts

```diff
@@ -18,9 +18,13 @@ export const formatStaking = (data: Staking, t: any) => {
     },
     {
       label: t('unbondingTime'),
-      detail: t('days', {
-        day: secondsToDays(nanoToSeconds(data.unbondingTime)),
-      }),
+      detail: data.unbondingTime > 1000000000 * 60 * 60 * 24 // nanoseconds of one day
+        ? t('days', {
+          day: secondsToDays(nanoToSeconds(data.unbondingTime)),
+        })
+        : t('seconds', {
+          second: secondsToDays(nanoToSeconds(data.unbondingTime)),
+        }),
```

Comment:
You are still using the same `secondsToDays` function for both `seconds` and `days`. Which doesn't make sense if you want to show the actual seconds for the seconds translation.

As I see this logic is duplicated a few times, it might make sense to make a util function and reuse it like so:

```
const convertBySeconds = (seconds, t) => {
  const SECONDS_IN_DAY = 86400;
  return (
    seconds >= SECONDS_IN_DAY ? (
      t('days', {
        day: secondsToDays(seconds),
      })
    ) : (
      t('seconds', {
        second: seconds,
      })
    )
  );
};
```

and in the following detail we can do:
```
// staking
{
  label: t('unbondingTime'),
  detail: convertBySeconds(nanoToSeconds(data.unbondingTime), t),
}

// gov
{
  label: t('maxDepositPeriod'),
  detail: convertBySeconds(nanoToSeconds(data.maxDepositPeriod), t),
}
```

I personally haven't tested this out yet though. 

I'm not sure if the other categories also need to be updated but I will leave that to you

Reply:
I've tested yesterday it was fine. But I'll revise with this suggestion and test more. Indeed repeating logic should do another util function! 

What do you mean by "other categories"? Do you mean same param page or other pages of explorer?